src/commands/env/create/scratch.ts

```diff
@@ -76,6 +76,7 @@ export default class EnvCreateScratch extends SfCommand<ScratchCreateResponse> {
     alias: Flags.string({
       char: 'a',
       summary: messages.getMessage('flags.alias.summary'),
+      description: messages.getMessage('flags.alias.description'),
```

Comment:
now that we're using a LOT of messages, line 25 either needs to include all the new description messages OR be changed to 
`const messages = Messages.loadMessages('@salesforce/plugin-env', 'create_scratch');`
(to just load them all instead of checking each one)

Reply:
Thx! And done!